rust/src/cxxrsutil.rs

```diff
@@ -74,45 +74,29 @@ macro_rules! impl_wrap {
     };
 }
 
-/// Custom macro to bind an OSTree GObject type.
-macro_rules! bind_ostree_obj {
-    ($w:ident) => {
+/// Custom macro to bind gtk-rs bridged types.
+macro_rules! cxxrs_bind {
+    ($ns:ident, $lowerns:ident, $sys:ident, [ $( $i:ident ),* ]) => {
```

Comment:
Minor: it feels a bit odd to see the standalone `$ns`.
It is always used in conjunction with the `$i` entries and results in some weird freestanding `G`.
Would you maybe consider dropping it and just have it repeated in each `$i`? Personally I think that it would improve readability, and the prefix duplication is not that terrible.